### ISSUE TITLE
add an endpoint for blocks per layer 

### ISSUE THREAD

**Issue by AmitShaul** [spacemeshos/api#135]
add a new endpoint to return the number of blocks created (by a single node) per layer

**Comment by lrettig** [comment]
Sounds like a simple query that takes one argument (a layerID) and returns an array of block objects

**Comment by AmitShaul** [comment]
yes, this is exactly what needed, adding it to NodeService sounds most reasonable with the services we have at the moment.

**Comment by avive** [comment]
You should add it to MeshServce - the NodeService is designed to give status of mesh syncing of current node and manage the node. The MeshService is designed to provide mesh data.
